<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MF_NGraph</class>
 <widget class="QWidget" name="MF_NGraph">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>632</width>
    <height>725</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>436</x>
     <y>600</y>
     <width>77</width>
     <height>98</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="pb_Test3">
      <property name="text">
       <string>tst3</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pb_Test2">
      <property name="text">
       <string>tst2</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pb_Test">
      <property name="text">
       <string>tst1</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QSlider" name="verticalSlider">
   <property name="geometry">
    <rect>
     <x>552</x>
     <y>312</y>
     <width>16</width>
     <height>160</height>
    </rect>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>748</y>
     <width>30</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>552</x>
     <y>96</y>
     <width>177</width>
     <height>177</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QFrame" name="frame_gr">
   <property name="geometry">
    <rect>
     <x>52</x>
     <y>48</y>
     <width>453</width>
     <height>353</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <widget class="QScrollBar" name="scr_h">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>248</y>
      <width>257</width>
      <height>16</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="pageStep">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QScrollBar" name="scr_v">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>20</y>
      <width>16</width>
      <height>197</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Gr_View" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>56</x>
      <y>32</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>76</x>
      <y>4</y>
      <width>245</width>
      <height>23</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>X</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="sb_ScaleX">
       <property name="minimum">
        <double>0.100000000000000</double>
       </property>
       <property name="maximum">
        <double>10.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.200000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Y</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="sb_ScaleY">
       <property name="minimum">
        <double>0.100000000000000</double>
       </property>
       <property name="maximum">
        <double>10.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.200000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Gr_View</class>
   <extends>QGraphicsView</extends>
   <header>gr_view.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>pb_Test</sender>
   <signal>clicked()</signal>
   <receiver>MF_NGraph</receiver>
   <slot>pb_Test_Clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>375</x>
     <y>344</y>
    </hint>
    <hint type="destinationlabel">
     <x>289</x>
     <y>320</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pb_Test2</sender>
   <signal>clicked()</signal>
   <receiver>MF_NGraph</receiver>
   <slot>pb_Test2_Clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>431</x>
     <y>302</y>
    </hint>
    <hint type="destinationlabel">
     <x>240</x>
     <y>189</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pb_Test3</sender>
   <signal>clicked()</signal>
   <receiver>MF_NGraph</receiver>
   <slot>pb_Test3_Clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>431</x>
     <y>262</y>
    </hint>
    <hint type="destinationlabel">
     <x>240</x>
     <y>189</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sb_ScaleX</sender>
   <signal>valueChanged(double)</signal>
   <receiver>MF_NGraph</receiver>
   <slot>sb_ScaleX_Changed(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>440</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>241</x>
     <y>351</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sb_ScaleY</sender>
   <signal>valueChanged(double)</signal>
   <receiver>MF_NGraph</receiver>
   <slot>sb_ScaleY_Changed(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>40</x>
     <y>350</y>
    </hint>
    <hint type="destinationlabel">
     <x>241</x>
     <y>351</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>pb_Test_Clicked()</slot>
  <slot>pb_Test2_Clicked()</slot>
  <slot>pb_Test3_Clicked()</slot>
  <slot>sb_ScaleX_Changed(double)</slot>
  <slot>sb_ScaleY_Changed(double)</slot>
 </slots>
</ui>
